Ginger/Ginger/ALM/Repository/QtestRepository.cs

```diff
@@ -298,6 +298,13 @@ public override bool ExportActivitiesGroupToALM(ActivitiesGroup activtiesGroup,
             //TODO: retireve test case fields -->DONE
             ObservableList<ExternalItemFieldBase> testCaseFields =  WorkSpace.Instance.Solution.ExternalItemsFields;
             var filterTestCaseFields = testCaseFields.Where(tc => tc.ItemType == eQCItemType.TestCase.ToString()).ToList();
+            for (var i = 0; i < activtiesGroup.ActivitiesIdentifiers.Count; i++)
```

Comment:
Instead of looping through all ActivitiesIdentifiers, add filter here itself to loop over only null element

Reply:
Added